.storybook/preview.js

```diff
@@ -56,51 +55,18 @@ const SORT_ORDER = {
 };
 
 export const parameters = {
-  layout: 'centered',
```

Comment:
I think that it makes sense to remove this, because the flex wrapper that this adds can cause issues when developing using Storybook (for example: https://github.com/sumup-oss/circuit-ui/issues/1312)

Reply:
Hm, one downside to this is that it looks like Chromatic takes a full-page screenshot, making the screenshots a bit less nice.

Thoughts on whether to keep or revert this?

Reply:
I personally prefer the centered view. If this causes an issue for a particular story, it can be overridden per story file.

Reply:
Okay, let's stick with the status quo (centered view) for now. If we run into other issues like #1312 we might want to rethink it—but basically a flex parent can potentially affect any block children